README.md

```diff
@@ -58,6 +58,98 @@ running Go 1.9 or higher.
 
 - [Getting Started with Azure Blob Service in Go](https://github.com/Azure-Samples/storage-blob-go-getting-started)
 
+# Generate code
+
+## Generate the SDK
```

Comment:
I'd like to see more distinctive section names, it will drive clarity for folks. Perhaps:

- _Generating a Go package from Swagger_
  This section would target AutoRest directly, passing the flag `--go` and little else. Mostly this would be helpful in emphasizing to the reader some of the basics. The role of AutoRest, and how it relates to our project. It may also be a helpful place to nod to the fact that in the Go SDK we expect that each API Version will be generated. We could also augment these instructions with running tools like `go vet`, `go lint`, etc.
- _Generating an Azure-SDK-for-Go service package_
  This is section could break down using AutoRest to target a particular flag, and output to the expected location in an Azure-SDK-for-Go fork. It's a good place to start introducing some more advanced Go package management concepts. In the future, it may be a good place to explore using `go dep` pointing at a fork instead of a main branch.
- _Generating Azure SDK for Go service packages in bulk_
 This section could introduce the `generator` script, which will sorta automatically create the packages as configured above, but which doesn't expose the ability to granularly generate a single tag. (This is also maybe good feedback for the `generator` tool.
- _Generating Azure SDK for Go profiles_
Here we could get in to defining a coherent list of API Versions and running the profile builder tool.

Each section would probably need to define its own set of pre-requisites :)

Reply:
Fixed